src/server/bin.js

```diff
@@ -25,6 +28,13 @@ async function run () {
   const libp2pMa = argv.libp2pMultiaddr || argv.lm || process.env.LIBP2PMA || '/ip6/::/tcp/5892/ws'
   const addresses = [multiaddr(libp2pMa)]
 
+  let peerInfo
+  if (argv.peerId) {
+    const peerData = fs.readFileSync(argv.peerId)
+    const peerId = await PeerId.createFromJSON(JSON.parse(peerData))
+    peerInfo = await PeerInfo.create(peerId)
+  }
```

Comment:
actually, maybe we should add a log here when the peer id is not provided, just to warn that a new id will be created at each restart, and recommend using the peer id option.